app/views/applications/new.html.haml

```diff
@@ -20,6 +20,62 @@
             = f.input :amount_other_sources, label: "Kwota z innych źródeł"
           .col-md-4
             = f.input :amount_overall, label: "Kwota ogółem"
+
+    .panel.panel-primary
+      .panel-heading Część II - informacje o projekcie
+      .panel-body
+        .row
+          = f.input :description, label: "Tematyka i opis projektu"
+        .row
+          %h4.text-center Planowany termin i miejsce (jeżeli dotyczy) realizacji projektu
+          .col-md-6
+            = f.input :location, label: "Miejsce"
+          .col-md-6 
+            = f.input :date, label: "Data"
         .row
+          = f.input :project_subject, label: "Cel projektu"
+          = f.input :required_resources, label: "Informacja o zasobach niezbędnych do realizacji projektu (lokalowych, technicznych, kadrowych) wraz z koncepcją ich pozyskania"
+          = f.input :past_schedule, label: "Wykaz realizacji wnioskowanego projektu w latach poprzednich"
+          = f.input :collaboration, label: "Współpraca organizacyjna z innymi podmiotami"
+          = f.input :target_group, label: "Docelowa grupa odbiorców projektu"
+          = f.input :sponsor_enlistment, label: "Opis strategii pozyskiwania sponsorów"
+          = f.input :promotion_plans, label: "Planowane działania promocyjne projektu"
+        .row
+          %h4.text-center Doświadczenie zespołu organizatorów w podobnych projektach:
+          = f.simple_fields_for :experiences do |ex|
+            .row
+              .col-md-4
+                = ex.input :project_name, label: "Nazwa Projektu"
+              .col-md-4
+                = ex.input :role, label: "Pełniona funkcja"
+              .col-md-4
+                = ex.input :budget, label: "Budżet"
+            .row
+              .col-md-6
+                = ex.input :project_date, label: "Data wydarzenia"
+              .col-md-6
+                = ex.input :user_id, collection: @users.map { |p| p.full_name }, label: "Imię i nazwisko członka zespołu"
```

Comment:
Instead of `@users.map { |p| p.full_name }` you can do `User.pluck(&:full_name)` instead if you really need names only.

Reply:
On a side note, is the name enough or should the form submit the id associated with the user?

Reply:
The database needs the id, the user needs the name shown, I'm pretty much sure that
`User.all.map { |u| [u.id, u.full_name] }` should work.